src/components/DataTable/TableExpandRow.js

```diff
@@ -79,6 +81,15 @@ TableExpandRow.propTypes = {
    * The description of the chevron right icon, to be put in its SVG `<title>` element.
    */
   expandIconDescription: PropTypes.string,
+
+  /**
+   * The id of the matching th node in the table head. Solves a11y DAP violation to match these appropriately.
```

Comment:
Since we don't have access to this DAP test or the referenced violation this should be changed to reference [1.3.1](https://www.ibm.com/able/guidelines/ci162/info_and_relationships.html).

Reply:
Added references into the comment here